shared/js/background/helpers/arguments-object.js

```diff
@@ -17,7 +17,6 @@ export function getArgumentsObject (tabId, sender, documentUrl, sessionKey) {
     // Clone site so we don't retain any site changes
     // @ts-ignore
     const site = tabClone.site
-    const referrer = tab?.referrer || ''
```

Comment:
We're no longer needing this in C-S-S so let's remove it as it wasn't ever documented or intended to be required.